### DISCUSSION TITLE
HACS install

### DISCUSSION THREAD

**Discussion by lordee** [firstof9/lovelace-grocy-meal-plan-card#6] [Q&A] [answered]
Hi, is this meant to be installable via HACS?  I can't see it when searching.

**Comment by firstof9** [comment, marked as answer]
Yes you can add it as a custom repo in HACS.

**Comment by NanKooDev** [comment]
Since the answer is not really that obvious for some, here's a more detailed instruction for future people:
1. Go to HACS in Home Assistant.
2. Click the 3 dots on the top right
3. Click `Custom Repositories`
4. In the field `repositories` enter the url of this repo `https://github.com/firstof9/lovelace-grocy-meal-plan-card`
5. Set the type to `Dashboard`
6. Click `Add`
7. Now search for `Grocy` and you'll see it pop up.
